deviceModeInit.js

```diff
@@ -46,10 +47,14 @@ var rudderTracking = (function () {
 
   function init() {
     pageCurrency = Shopify.currency.active;
+    userId = ShopifyAnalytics.meta.page.customerId || __st.cid;
```

Comment:
In the checkout pages, I am seeing `__st.cid` is present while `ShopifyAnalytics.meta.page.customerId` is undefined. 
Point to note though, server-side calls for tracking should work just fine.

Reply:
It should be good, once we have captured `userId` it will be stored as a cookie for Rudderstack sdk, So we should be good